talent/application-process.md

```diff
@@ -0,0 +1,69 @@
+# Applying at source{d} 
+
+This document describes how the developers are able to apply at source{d} and the hiring process. 
```

Comment:
This document describes the source{d} hiring process from the point of view of the candidate.